dev/com.ibm.ws.anno_fat/fat/src/com/ibm/ws/tests/anno/caching/CacheEnablementTest.java

```diff
@@ -168,7 +168,7 @@ public void cacheEnablement_testEnabledCache() throws Exception {
         //         Verify that the cache is created.
 
         installServerXml("jandexDefaultsAutoExpandTrue_server.xml");
-        installJvmOptions("JvmOptions_Enabled.txt");
+        installJvmOptions("JvmOptions_Enabled_Text.txt");
```

Comment:
Why do we even test any other format than binary?  I thought you would have removed the other two options.

Reply:
That test was written when text options were the main format, and do a text update to one of the cache files.  I haven't yet gotten to rewriting the text to make the same update to the binary format file.

Reply:
I was surprised it was even an option to do text or jandex format.  I figured all that code to create text or jandex format would have been deleted and not part of this pull request any longer.